pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -112,7 +112,7 @@
 import org.apache.pinot.segment.local.utils.TableConfigUtils;
 import org.apache.pinot.spi.config.table.SegmentsValidationAndRetentionConfig;
 import org.apache.pinot.spi.config.table.TableConfig;
-import org.apache.pinot.spi.config.table.TableStats;
+import org.apache.pinot.spi.config.table.TableStatsHumanReadable;
```

Comment:
Consider moving this change (TableStats -> TableStatsHumanReadable) to a separate PR for cleaner rollbacks / cherry-picks.

Reply:
I had to do it in this PR because I'm making use of TableStats and there's a dependency.